external/os-guides/pentesting/kali.md

```diff
@@ -6,595 +6,156 @@ redirect_from:
-        [user@dom0 ~]$ qvm-shutdown debian-<X>
-
-3. Clone `debian-X` template
-
-        [user@dom0 ~]$ qvm-clone debian-<X> kali-rolling
-
-4. Check the name of currently used repository in `/etc/apt/sources.list` and current testing [Debian release][Debian-releases]. Update repository list accordingly
-
-        [user@kali-rolling ~]$ sudo sed -i 's/<current stable>/<current testing>/g' /etc/apt/sources.list
-        [user@kali-rolling ~]$ sudo sed -i 's/<current stable>/<current testing>/g' /etc/apt/sources.list.d/qubes-r<X>.list
-      
-      e.g. in this example we update `stretch` stable repository to `buster` testing repository
-      
-        [user@kali-rolling ~]$ sudo sed -i 's/stretch/buster/g' /etc/apt/sources.list
-        [user@kali-rolling ~]$ sudo sed -i 's/stretch/buster/g' /etc/apt/sources.list.d/qubes-r<X>.list
-        
-For installation based on Debian 10 stable, please note that the security repository of Debian testing has [recently been renamed][Debian-security-naming-convention] from `<current testing>/update` to `<current-testing>-security`. To account for that change, execute the following command.
-
-        [user@kali ~]$ sudo sed -i 's/bullseye\/updates/bullseye-security/g' /etc/apt/sources.list
-
-5. Upgrade `kali-rolling` template to latest Debian testing release
-
-        [user@kali-rolling ~]$ sudo apt-get update
-        [user@kali-rolling ~]$ sudo apt-get dist-upgrade
-        [user@kali-rolling ~]$ sudo apt-get autoremove
-
-**Note:** During execution of a `dist-upgrade` command read carefully list of packages to be removed. If it contains `qubes-*` packages, terminate operation and try to resolve `qubes-*` packages missing dependencies first.
-
-6. Copy the Kali GPG key from the DispVM to the new template:
-      
-        [user@xxxx-dvm ~]$ qvm-copy kali-key.asc
-
-   The DispVM can now be turned off.
-
-7. Add the Kali GPG key to the list of keys trusted to authenticate packages:
-
-        [user@kali-rolling ~]$ cat /home/user/QubesIncoming/dispXXX/kali-key.asc | sudo apt-key add -
-
-   This command should return `OK` on a line by itself.
-
-8. Attempt the upgrade process in the new template.
-
-        [user@kali-rolling ~]$ sudo cat <<EOF > /etc/apt/sources.list.d/kali.list
-        # Kali Linux repository
-        deb http://http.kali.org/kali kali-rolling main non-free contrib
-        EOF
-        [user@kali-rolling ~]$ sudo apt-get update
-        [user@kali-rolling ~]$ sudo apt-get dist-upgrade
-        [user@kali-rolling ~]$ sudo apt-get autoremove
-
-9. Ensure a terminal can be opened in the new template.
-
-        [user@dom0 ~]$ qvm-run -a kali-rolling gnome-terminal
+10. Ensure a terminal can be opened in the new template.
 
-#### Install the Kali tools
+```shell_session
+$ qvm-run -a kali-rolling gnome-terminal
+```
 
+Install the Kali tools
+------------------------------
 At this point you should have a working template and you can install the tools you need.
```

Comment:
I *would* at least point to the available metapackages here.

Reply:
Ok, added a link to the official doc